src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseSignature.cs

```diff
@@ -241,6 +241,43 @@ public bool VerifyDetached(AsymmetricAlgorithm key, byte[] detachedContent, byte
             return VerifyCore(coseKey, detachedContent, null, associatedData);
         }
 
+        /// <summary>
+        /// Verifies that the signature is valid for the message's content using the specified key.
+        /// </summary>
+        /// <param name="key">The private key used to sign the content.</param>
```

Comment:
Looks like all of these Verify overloads demand the private key.  Since they're all wrong, again, just leave this matching... but we should fix these at some point.